jinahub/indexers/storage/RedisStorage/tests/conftest.py

```diff
@@ -0,0 +1,25 @@
+from jina import Document, DocumentArray
+import pytest
+
+from redis_storage import RedisStorage
+from pytest_mock_resources import create_redis_fixture
+
+redis = create_redis_fixture()
```

Comment:
I don't quite get what this does. Is this a mocked redis? 

Can u instead use a docker-compose, like we do in PSQL and Mongo? https://github.com/jina-ai/executors/blob/11a488acb56865f96a7816d2d9b915a5ff0c2041/jinahub/indexers/storage/PostgreSQLStorage/tests/test_postgres_dbms.py#L100

Reply:
yes tthis mocks redis and provides a test hostname and port. Thought it would be better than using docker.
I'll modify it to use docker-compose

Reply:
done